### DISCUSSION TITLE
Why is rustDesk more expensive than AnyDesk ?

### DISCUSSION THREAD

**Discussion by Extarys** [rustdesk/rustdesk#7201] [Q&A] [answered]
I was thinking about purchasing a license or something simply to have a preconfigured client. I realized I can take an hour and a half just trying to get my grandma to check her emails for the rustdesk client download link (and the server url and the key). She thinks "Google"" is the computer and the window is her monitor, so imagine that. :P

I also had a friend drive 2 hours to my house because she have computer anxiety and couldn't figure out where the downloaded file went (to be fair we would still have the same issue with the preconfigured client, but again, imagine the steps after that over the phone).

I do make some paid remote support too, but my few customers usually can install and configure Rustdesk fairly easily. 

Then I saw that AnyDesk offer their Remote Support for way less depending on how many devices you need. And you don't "need" to self host (ie: pay server fee and maintain it). I was under the impression RustDesk would be way cheaper :/ I love having a self-hosted remote support app, but still.

![image](https://github.com/rustdesk/rustdesk/assets/11581433/1baaeed3-0287-4606-ae95-c8dcb88cb000)
![image](https://github.com/rustdesk/rustdesk/assets/11581433/f376724e-5616-4704-a2e3-a7e49195d475)
![image](https://github.com/rustdesk/rustdesk/assets/11581433/35e4bb81-cf34-4e6c-b62c-ab534193edf3)

Am I missing something ? (Not a harsh tone here, just a bit surprised/disappointed).

**Comment by rustdesk** [comment, marked as answer]
You should compare with AnyDesk's self-hosted pricing. We did contact their sales about their self-hosted pricing.

I totally understand normal users do not care about if it is self-hosted. Actually, self-hosted is mainly for enterprise users who care about privacy / performance more, that's why it is more expensive. I also understand the logic of `self-hosted should be cheaper`, if I am the consumer, I will also think so. But actually AnyDesk's self-hosted is much expensive than SaaS, we just follow the game rule.

BTW, AnyDesk's SaaS just looks cheap, they give you a lot of users and devices, but actually they limit you by concurrent connections, only 1 connection is allowed in the 20/500 plan. Maybe you do not care about this limit, but actually most of clients care about this very much, this is also the most frequent question they ask about us.

Before our SaaS is ready, we can only earn cash with current self-hosted to maintain the dev team.

**Reply by Extarys** [reply]
I love self-hosted though ^^ I didn't know AnyDesk had a self-hosted option so my comparison above is apple to oranges.

> AnyDesk's SaaS just looks cheap, they give you a lot of users and devices, but actually they limit you by concurrent connection

I also didn't notice the concurrent connection limit. Yikes. Although I'm alone, I understand SMB would like more than 1 connection.


Thanks for enlightening me and for your fast reply.
